src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/Currency.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ This file is part of the Sylius package.
+ (c) Sylius Sp. z o.o.
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Currency\Model\Currency" shortName="Currency">
+        <validationContext>
+            <values>
+                <value name="groups">
+                    <values>
+                        <value>sylius</value>
+                    </values>
+                </value>
+            </values>
+        </validationContext>
+
+        <operations>
+            <operation class="ApiPlatform\Metadata\Get"  uriTemplate="/admin/currencies/{code}">
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>admin:currency:read</value>
+                            </values>
+                        </value>
+                    </values>
+                </normalizationContext>
+            </operation>
+
+            <operation class="ApiPlatform\Metadata\GetCollection" uriTemplate="/admin/currencies">
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>admin:currency:read</value>
```

Comment:
`read` needs to be split into `show` and `index`, please check with @lchrusciel

Reply:
Yes, we are aware of this. In this PR, we are not splitting `read` into `show` and `index`.